tests/test_cli_workflows.py

```diff
@@ -68,7 +72,9 @@ def test_workflows_server_ok():
             "reana_client.api.client.current_rs_api_client",
             make_mock_api_client("reana-server")(mock_response, mock_http_response),
         ):
-            result = runner.invoke(cli, ["list", "-t", reana_token])
```

Comment:
Could you add another test case that covers the call without `--include-progress`, and maybe also another for `--include-workspace-size`?